README.md

```diff
@@ -31,11 +31,10 @@ Latest installers with Python 3.9 `(*)` in the base environment:
 
 `(*)` The Python version is specific only to the base environment. Conda can create new environments with different Python versions and implementations.
 
-`(**)` While the Raspberry PI includes a 64 bit processor, the
-[RasbianOS](https://www.raspberrypi.org/software/operating-systems/) is built
-on a 32 bit kernel and is not a supported configuration for these installers.
-We recommend using a 64 bit linux distribution such as [Ubuntu for Raspberry
-PI](https://ubuntu.com/raspberry-pi).
+`(**)` The Raspberry PI includes a 64 bit processor, but you must also use the
+full 64-bit version of [Raspberry Pi OS 64-bit](https://www.raspberrypi.com/software/operating-systems/#raspberry-pi-os-64-bit), NOT the versions that say "System: 32-bit". If a
+64-bit version of Raspberry Pi OS is not available for your hardware version, we
+recommend using a 64 bit linux distribution such as [Ubuntu for Raspberry PI](https://ubuntu.com/raspberry-pi).
```

Comment:
I personally do not want to recommend anything since we are not compatible with 32 bit os. Can you please say something like:

"I a 64-bit version of Raspberry Pi OS is not available for your hardware, the installers provided here are not compatible with your hardware."

Reply:
Currently the docs recommend using Ubuntu for RPi in order to get a 64bit OS on RPi. We want to remove that information?

Reply:
I guess I misunderstood your logic.

The original sentence was:

1. Constraints
2. Recommendation
3. Explicitly calling out a limitation
4. Other recommendation

I reworded it as:
1. Constraint
2. Recommendations
3. Explicitly calling out a limitation